.github/workflows/publish_release.yml

```
@@ -215,37 +91,44 @@ jobs:
           git_config_global: true
           git_user_signingkey: true
           git_commit_gpgsign: true
-          git_tag_gpgsign: false
-
-      - name: Setup Java
-        uses: actions/setup-java@3a4f6e1af504cf6a31855fa899c6aa5355ba6c12 # v4.7.0
-        with:
-          distribution: ${{ inputs.java-distribution || 'temurin' }}
-          java-version: ${{ inputs.java-version || '21.0.4' }}
-
-      - name: Setup Gradle
-        uses: gradle/actions/setup-gradle@94baf225fe0a508e581a564467443d0e2379123b # v4.3.0
-        with:
-          gradle-version: wrapper
+          git_tag_gpgsign: true
```

Will possibly break semantic release since it uses lightweight tags.

```suggestion
          git_tag_gpgsign: false
```